### PR TITLE
Cfae adding new essentia features

### PR DESCRIPTION
**Pull request by Casperfrc**

### CODE REVIEW

File: src/loudness/loudness_extractor.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    b.momentaryLoudness >> (p, "momentaryLoudness")
+    b.shortTermLoudness >> (p, "shortTermLoudness")
+    b.integratedLoudness >> (p, "integratedLoudness")
+    b.loudnessRange >> (p, "loudnessRange")
+
+    run(audio)
+
+    max_loudness = (max(p["momentaryLoudness"]) + max(p["shortTermLoudness"])) / 2
+
+    return max_loudness, p["integratedLoudness"], p["loudnessRange"]
```
Comment (Areedtz) [inline src/loudness/loudness_extractor.py]: Missing empty line at the end of the file

File: src/loudness/t/test_loudness_extractor.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+from utilities.filehandler.audio_loader import get_audio_loaded_song
+
+def test_get_song_loudness():
+    song = get_audio_loaded_song("loudness/t/test_loudness_extractor/8376-1-"
+                               + "1_Demolition_Man_proud_music_preview.wav")
+    max_loudness, integratedLoudness, loudnessRange = get_song_loudness(song)
+
+    print(max_loudness)
+    print(integratedLoudness)
+    print(loudnessRange)
```
Comment (Areedtz) [inline src/loudness/t/test_loudness_extractor.py]: Should probably remove these prints

File: src/utilities/filehandler/audio_loader.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    -------
+    stereosample
+        The input stereo audio signal
+
+    """
+
+    path = get_absolute_path(song_path)
+    loader = AudioLoader(filename=path)
+
+    return loader
```
Comment (Areedtz) [inline src/utilities/filehandler/audio_loader.py]: Missing empty line at the end of the file

File: src/classification/classifier/profile_data_extractor.py [hunk 4]
```diff
@@ -73,4 +72,4 @@ def get_classifier_data(
          (mood_sad, mood_sad_probability)
          ]
 
-    return t[0], t[1], t[2], t[3], t[4], t[5]
+    return t[0], t[1], t[2], t[3], t[4], t[5]
```
[the remark this answers is not in the document]
Comment (Casperfrc) [reply src/classification/classifier/profile_data_extractor.py]: +1

File: src/classification/classifier/profile_data_extractor.py [hunk 5]
```diff
@@ -8,7 +8,6 @@
 
 from utilities.filehandler.handle_path import get_absolute_path
 
-
```
[the remark this answers is not in the document]
Comment (Casperfrc) [reply src/classification/classifier/profile_data_extractor.py]: +1

File: src/loudness/loudness_extractor.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
+import sys
+
+from typing import Tuple
```
[the remark this answers is not in the document]
Comment (Casperfrc) [reply src/loudness/loudness_extractor.py]: +1

File: src/loudness/loudness_extractor.py (hunk 1 above)
Comment (Casperfrc) [reply src/loudness/loudness_extractor.py]: +1

File: src/loudness/t/test_loudness_extractor.py (hunk 2 above)
Comment (Casperfrc) [reply src/loudness/t/test_loudness_extractor.py]: +1

File: src/utilities/filehandler/audio_loader.py (hunk 3 above)
Comment (Casperfrc) [reply src/utilities/filehandler/audio_loader.py]: +1

**Pull request merged**